autogpt_platform/frontend/src/app/signup/actions.ts

```diff
@@ -38,7 +38,7 @@ export async function signup(values: z.infer<typeof signupFormSchema>) {
       }
       console.log("Signed up");
       revalidatePath("/", "layout");
-      redirect("/marketplace/profile");
+      redirect("/");
```

Comment:
> After signing up, users are now redirected directly to their profile page instead of the homepage. This helps new users immediately see and edit their profile if needed.

Wait, Isn't this the reverse?